RUNNING: "python -m distributed.cli.dask_scheduler --protocol=tcp
                    --scheduler-file /root/cugraph/python/cugraph-pyg/cugraph_pyg/examples/scheduler.json
                "
/opt/conda/envs/rapids/lib/python3.10/site-packages/distributed/cli/dask_scheduler.py:140: FutureWarning: dask-scheduler is deprecated and will be removed in a future release; use `dask scheduler` instead
  warnings.warn(
2023-04-04 14:48:56,895 - distributed.scheduler - INFO - -----------------------------------------------
2023-04-04 14:48:57,571 - distributed.scheduler - INFO - State start
2023-04-04 14:48:57,585 - distributed.scheduler - INFO - -----------------------------------------------
2023-04-04 14:48:57,586 - distributed.scheduler - INFO -   Scheduler at:  tcp://10.120.104.11:8786
2023-04-04 14:48:57,586 - distributed.scheduler - INFO -   dashboard at:  http://10.120.104.11:8787/status
2023-04-04 14:49:08,546 - distributed.scheduler - INFO - Register worker <WorkerState 'tcp://10.120.104.11:46529', status: init, memory: 0, processing: 0>
2023-04-04 14:49:08,548 - distributed.scheduler - INFO - Starting worker compute stream, tcp://10.120.104.11:46529
2023-04-04 14:49:08,548 - distributed.core - INFO - Starting established connection to tcp://10.120.104.11:35104
2023-04-04 14:49:08,550 - distributed.scheduler - INFO - Register worker <WorkerState 'tcp://10.120.104.11:44871', status: init, memory: 0, processing: 0>
2023-04-04 14:49:08,550 - distributed.scheduler - INFO - Starting worker compute stream, tcp://10.120.104.11:44871
2023-04-04 14:49:08,550 - distributed.core - INFO - Starting established connection to tcp://10.120.104.11:35116
2023-04-04 14:49:25,390 - distributed.scheduler - INFO - Receive client connection: Client-e4af8670-d2f7-11ed-84e7-5cff35c1a711
2023-04-04 14:49:25,390 - distributed.core - INFO - Starting established connection to tcp://10.120.104.11:60002
2023-04-04 14:49:25,636 - distributed.worker - INFO - Run out-of-band function '_func_set_scheduler_as_nccl_root'
2023-04-04 14:49:26,226 - distributed.scheduler - INFO - Receive client connection: Client-e4efe4e5-d2f7-11ed-84e6-5cff35c1a711
2023-04-04 14:49:26,227 - distributed.core - INFO - Starting established connection to tcp://10.120.104.11:60016
2023-04-04 14:49:26,878 - distributed.worker - INFO - Run out-of-band function '_func_set_scheduler_as_nccl_root'
2023-04-04 14:49:52,418 - distributed.scheduler - INFO - Remove client Client-e4efe4e5-d2f7-11ed-84e6-5cff35c1a711
2023-04-04 14:49:52,418 - distributed.core - INFO - Received 'close-stream' from tcp://10.120.104.11:60016; closing.
2023-04-04 14:49:52,418 - distributed.scheduler - INFO - Remove client Client-e4efe4e5-d2f7-11ed-84e6-5cff35c1a711
2023-04-04 14:49:52,419 - distributed.scheduler - INFO - Close client connection: Client-e4efe4e5-d2f7-11ed-84e6-5cff35c1a711
2023-04-04 14:49:52,518 - distributed.scheduler - INFO - Remove client Client-e4af8670-d2f7-11ed-84e7-5cff35c1a711
2023-04-04 14:49:52,519 - distributed.core - INFO - Received 'close-stream' from tcp://10.120.104.11:60002; closing.
2023-04-04 14:49:52,519 - distributed.scheduler - INFO - Remove client Client-e4af8670-d2f7-11ed-84e7-5cff35c1a711
2023-04-04 14:49:52,519 - distributed.scheduler - INFO - Close client connection: Client-e4af8670-d2f7-11ed-84e7-5cff35c1a711
2023-04-04 14:50:10,725 - distributed._signals - INFO - Received signal SIGINT (2)
2023-04-04 14:50:10,725 - distributed.scheduler - INFO - Scheduler closing...
2023-04-04 14:50:10,725 - distributed.scheduler - INFO - Scheduler closing all comms
2023-04-04 14:50:10,726 - distributed.core - INFO - Connection to tcp://10.120.104.11:35104 has been closed.
2023-04-04 14:50:10,726 - distributed.scheduler - INFO - Remove worker <WorkerState 'tcp://10.120.104.11:46529', status: running, memory: 0, processing: 0>
2023-04-04 14:50:10,726 - distributed.core - INFO - Removing comms to tcp://10.120.104.11:46529
2023-04-04 14:50:10,726 - distributed.core - INFO - Connection to tcp://10.120.104.11:35116 has been closed.
2023-04-04 14:50:10,726 - distributed.scheduler - INFO - Remove worker <WorkerState 'tcp://10.120.104.11:44871', status: running, memory: 0, processing: 0>
2023-04-04 14:50:10,727 - distributed.core - INFO - Removing comms to tcp://10.120.104.11:44871
2023-04-04 14:50:10,727 - distributed.scheduler - INFO - Lost all workers
2023-04-04 14:50:10,727 - distributed.batched - INFO - Batched Comm Closed <TCP (closed) Scheduler connection to worker local=tcp://10.120.104.11:8786 remote=tcp://10.120.104.11:35104>
Traceback (most recent call last):
  File "/opt/conda/envs/rapids/lib/python3.10/site-packages/distributed/comm/tcp.py", line 317, in write
    raise StreamClosedError()
tornado.iostream.StreamClosedError: Stream is closed

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/conda/envs/rapids/lib/python3.10/site-packages/distributed/batched.py", line 115, in _background_send
    nbytes = yield coro
  File "/opt/conda/envs/rapids/lib/python3.10/site-packages/tornado/gen.py", line 769, in run
    value = future.result()
  File "/opt/conda/envs/rapids/lib/python3.10/site-packages/distributed/comm/tcp.py", line 328, in write
    convert_stream_closed_error(self, e)
  File "/opt/conda/envs/rapids/lib/python3.10/site-packages/distributed/comm/tcp.py", line 144, in convert_stream_closed_error
    raise CommClosedError(f"in {obj}: {exc}") from exc
distributed.comm.core.CommClosedError: in <TCP (closed) Scheduler connection to worker local=tcp://10.120.104.11:8786 remote=tcp://10.120.104.11:35104>: Stream is closed
2023-04-04 14:50:10,728 - distributed.batched - INFO - Batched Comm Closed <TCP (closed) Scheduler connection to worker local=tcp://10.120.104.11:8786 remote=tcp://10.120.104.11:35116>
Traceback (most recent call last):
  File "/opt/conda/envs/rapids/lib/python3.10/site-packages/distributed/comm/tcp.py", line 317, in write
    raise StreamClosedError()
tornado.iostream.StreamClosedError: Stream is closed

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/conda/envs/rapids/lib/python3.10/site-packages/distributed/batched.py", line 115, in _background_send
    nbytes = yield coro
  File "/opt/conda/envs/rapids/lib/python3.10/site-packages/tornado/gen.py", line 769, in run
    value = future.result()
  File "/opt/conda/envs/rapids/lib/python3.10/site-packages/distributed/comm/tcp.py", line 328, in write
    convert_stream_closed_error(self, e)
  File "/opt/conda/envs/rapids/lib/python3.10/site-packages/distributed/comm/tcp.py", line 144, in convert_stream_closed_error
    raise CommClosedError(f"in {obj}: {exc}") from exc
distributed.comm.core.CommClosedError: in <TCP (closed) Scheduler connection to worker local=tcp://10.120.104.11:8786 remote=tcp://10.120.104.11:35116>: Stream is closed
2023-04-04 14:50:10,729 - distributed.scheduler - INFO - Stopped scheduler at 'tcp://10.120.104.11:8786'
2023-04-04 14:50:10,729 - distributed.scheduler - INFO - End scheduler
